src/book/03-guides/01-main-codebase/02-tuning-scripts.mdx

```diff
@@ -0,0 +1,137 @@
+---
+section: Guides
+chapter: Main Codebase
+title: Running and Tuning Scripts
+description: How to tune and run scripts.
+slug: /guides/main/script-tuner
+---
+
+Scripts are static motions for the robot. They specify what joint angles to move to and how long the robot should take to go to those joint positions. For example, standing up is a small script telling the robot to move their joints to the stand position over one second. There can be many of these position specifications in a row to make the robot do more complex movements like getting up or kicking.
```

Comment:
`static motions` is an oxymoron. I know what you're trying to say, but I am not sure if someone who this was aimed at would.

Reply:
Nah, it's a thing. In contrast to dynamic motions:

- Static motions: motions that don't change at runtime
- Dynamic motions: motions that change at runtime, perhaps in response to things like changing balance, stability, etc

Reply:
When I google the term I don't get anything related on the first page, except the [statics wikipedia page](https://en.wikipedia.org/wiki/Statics) comes up, which is certainly not what this is referring to. I know exactly what you're saying, but I think we could use better words that `static motion` for it - maybe even add a link to those words to some reference material defining it the way you have, if you like those words.

Reply:
We could follow up the first sentence with "By static, we mean they are preconfigured motions that won't change at runtime."

Reply:
Or just replace the first sentence with "Scripts are preconfigured motions for the robot that don't change at runtime.".

Reply:
Sure. I think anything like that which explains that we are saying that the movements are preconfigured will do the trick fine.

Reply:
Probs worth putting static/dynamic motion into the glossary then if they're not common terms. We use them a lot.